CalibCalorimetry/EcalTPGTools/plugins/EcalTPGParamBuilder.h

```diff
@@ -1,6 +1,12 @@
 #ifndef ECALTPGPARAMBUILDER_H
 #define ECALTPGPARAMBUILDER_H
 
+//Author: Pascal Paganini - LLR
+//Date: 2006/07/10 15:58:06 $
+
+#define CMSSW_VERSION 340
```

Comment:
Added ?? No, it was the original version since many years.  As there were multiple commits in this PR, maybe I removed it and reused an older version after..
In any case, I will resubmit a new .h and .cc  without this if 340 (hoping there will be no consequence but I don't think so)  and resolve the double for loop in the .cc.
Thanks.